spec/Cabin/SingleSeat.vspec

```diff
@@ -53,14 +53,12 @@ Massage:
   description: Seat massage level. 0 = off. 100 = max massage.
 
 Recline:
-  datatype: int8
+  datatype: float
   type: actuator
-  min: -90
-  max: 90
   unit: degrees
-  description: Recline level. -90 = Max forward recline. 90 max backward recline.
+  description: Backrest recline compared to vertical axis. Negative degrees for forward recline. Positive degrees for backward recline.
 
-Position:
+Length:
```

Comment:
I would strongly prefer "position" over "length" (even if not fully definite), but length is something completely different. The length would be the length of the seating (the surface where one is sitting upon with his bottom) or the length of the backrest.

Reply:
The main problem I see with "position" is that VSC use it to describe all aspects of a seat position like in the struct below. They use the term "base" for the seating. If we keep "position" for "horizontal position of seating" then I think we better shall rename `position_t`in VSC, maybe to `seat_settings_t`or similar. 


```

    structs:
      #
      # Position of seat
      #
      - name: position_t
        description: |
          The position of the entire seat
        members:
          - name: base
            datatype: movement_t
            description: |
              The position of the base 0 front, 1000 back
          - name: cushion
            datatype: movement_t
            description: |
              The position of the cushion 0 short, 1000 long
          - name: lumbar
            datatype: movement_t
            description: |
              The position of the lumbar support
          - name: side_bolster
            datatype: movement_t
            description: |
              The position of the side bolster
          - name: head_restraint
            datatype: movement_t
            description: |
              The position of the head restraint 0 down, 1000 up
```

Reply:
Yes, "position_t" isn't a good name for all what's part of it. 
"Settings" would be more something you could set (like an actuator). Maybe "states" - but then if it also contains setting ...
"Properties" does also not fit well - that would be more like constants like "seat length", "set of controllable seat elements", ...
What about "state(s)_t", "seat_state(s)_t"?